index.html

```diff
@@ -33,6 +33,127 @@ <h2 class="name">I’m Ali<br>Glad to see you!</h2>
             </ul>
         </div>
       </article>
+      <!-- here we start the second milestone -->
+      <section class="grid-container">
+        <article class="middle tonic">
+          <h1 class="middle-h1">Tonic</h1>
+          <div class="canopy">
+            <span class="canopy-1">CANOPY</span>
+            <img src="images/Counter.png" alt="">
+            <span class="canopy-2">Back End Dev</span>
+            <img src="images/Counter.png" alt="">
+            <span class="canopy-2">2015</span>   
+          </div>
+          <p class="middle-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <img src="images/Snapshoot-Portfolio.png" alt="Snapshoot Portfolio" class="middle-images">
+          <ul class="middle-button">
+            <li class="html-button">html</li>
+            <li class="css-button">css</li>
+            <li class="js-button">javaScript</li>
+          </ul>
+          <button type="button" class="button-see-project">See Project</button>
+        </article>
```

Comment:
- [ ] The previous code review pointed out these sections specifically and suggested that all projects be in a `<ul>` tag and each project be in an `<li>` tag. As this was not optional, let's kindly refactor it. Remember this is for all projects listed.

Reply:
Thank you for your explanation, I will solve the issue.

Reply:
Well resolved